CHANGELOG.md

```diff
@@ -12,9 +12,12 @@
 - changed: Change FlipInput styling to make the edit functionality more obvious
 - changed: Enable max spend for Filecoin
 - changed: Move asset-specific settings into their own settings page
-- changed: Experiment config probability distribution support percentage based values
+- changed: Experiment config probability distribution support percentage based
+  values
```

Comment:
We don't allow newlines in our changelog entries, since that will screw up any attempt to sort or auto-merge. Each entry is its own line, no matter how long it is.

Reply:
I discussed with Sam on this and the new approach involves being smarter and parsing the contents of the entries along with their tags but we don't have it yet. I'll look into making the change to our gitattributes to handle this nicer later.